python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -394,23 +457,32 @@ def doIngestCalibs(repo, calib_repo, calibdatafiles, defectfiles):
 
     NOTE:
     calib ingestion ingests *all* the calibs, not just the ones needed
-    for the specified visits. We may want to revisit this in the future.
+    for certain visits. We may want to ...revisit... this in the future.
```

Comment:
I'm ...confused...

Reply:
This change isn't particularly meaningful, it's just me being clever with the word visit :)
The gist of the comment is that the ingestion steps will ingest everything in the provided raw (and calib) dir, not just a subset of images as might be desired. I'm not sure how easy (or desirable) it would be to search for visit info in, e.g., FITS headers, and only ingest those images (and their associated calibs).